drivers/entropy/entropy_stm32.c

```diff
@@ -377,6 +389,7 @@ static int entropy_stm32_rng_init(const struct device *dev)
 	 *  Linear Feedback Shift Register
 	 */
 	 LL_RCC_SetRNGClockSource(LL_RCC_RNG_CLKSOURCE_PLLSAI1);
+#endif
```

Comment:
For an easy match with #if: 
```suggestion
#endif // CONFIG_CLOCK_STM32_PLL_SRC_MSI && !CONFIG_CLOCK_STM32_PLL_SRC_HSI) \
&& !(CONFIG_CLOCK_STM32_PLL_SRC_HSE
```